packages/faustwp-cli/utils/validateFaustEnvVars.ts

```diff
@@ -1,4 +1,5 @@
 import { errorLog, warnLog } from './log.js';
+import { getWpSecret } from '../../faustwp-core/src/lib/getWpSecret.js';
```

Comment:
May need to address this too. To my point above, not sure if we can import from outside the scope of this package.